packages/template-retail-react-app/app/pages/cart/index.jsx

```diff
@@ -332,7 +332,7 @@ const Cart = () => {
                         >
                             <GridItem>
                                 <Stack spacing={4}>
-                                    {basket.productItems.map((productItem, idx) => {
+                                    {basket.productItems?.map((productItem, idx) => {
```

Comment:
l'm seeing the error: 

```
TypeError: Cannot read properties of undefined (reading 'map')
    at Cart (http://localhost:3000/mobify/bundle/development/pages-cart.js:430:26)
 ```
 
 **STR**
1. Running the project locally.
2. Load the empty cart page http://localhost:3000/global/en-GB/cart

Reply:
Correction: (I was using the wrong branch) I see a cart with 0 items, whereas in `develop` we show the empty cart place holder:

👉 https://pwa-kit.mobify-storefront.com/global/en-GB/cart

Reply:
That's the error this is fixing! 😁